swift_browser_ui_frontend/src/common/lang.js

```diff
@@ -526,24 +530,28 @@ let default_translations = {
                 "epäonnistunut salausohjelman lataus tai tilan loppuminen. " +
                 "Päivitä sivu ja kokeile uudelleen.",
         files: "Tiedostot",
-        ephemeral: "Tilapäinen yksityinen avain",
+        ephemeral: "Käyttää omaa yksityistä avainta",
+        multipleReceivers: "Lisää muita vastaanottimien julkisia avaimia",
         pk: "Yksityinen avain",
         pk_msg: "Lähettäjän yksityinen avain",
         phrase: "Yksityisen avaimen salasana",
         phrase_msg: "Yksityisen avaimen salasana",
         addkey: "Lisää vastaanottajan julkinen avain",
         addFiles: "Lisää tiedostoja",
         pubkey: "Vastaanottajien julkiset avaimet",
-        pubkeyLabel: "Julkinen avain",
+        pubkeyLabel: "Julkinen avain (sha256)",
         pubkey_msg: "Liitä vastaanottajan julkinen avain",
-        container: "Säiliö",
+        noRecipients: "Ei ylimääräisiä vastaanottimien julkisia avaimia",
+        container: "Kohteen Säiliö",
         container_msg: "Kohdesäiliö",
-        dropMsg: "Klikkaa lisätäksesi tiedostoja lähetettäväksi.",
+        container_hint: "Käytä tätä kenttää muuttaaksesi säiliön nimeä",
```

Comment:
Probably better as following, though finding a good general solution is a bit difficult:
Voit muuttaa säiliön nimeä tällä kentällä